templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  <ol>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>

```

Comment:
The links in the sidebar are placeholders. Replace `href="#"` with actual URLs using Django's `{% url %}` template tag to link to the correct views in your application.